starboard/extension/player_set_max_video_input_size.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_EXTENSION_PLAYER_SET_MAX_VIDEO_INPUT_SIZE_H_
+#define STARBOARD_EXTENSION_PLAYER_SET_MAX_VIDEO_INPUT_SIZE_H_
+
+#include <stdint.h>
+
+#include "starboard/drm.h"
+#include "starboard/media.h"
+#include "starboard/player.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kStarboardExtensionPlayerSetMaxVideoInputSizeName \
+  "dev.starboard.extension.PlayerSetMaxVideoInputSize"
+
+typedef struct StarboardExtensionPlayerSetMaxVideoInputSizeApi {
+  // Name should be the string
+  // |kStarboardExtensionPlayerSetMaxVideoInputSizeName|. This helps to validate
+  // that the extension API is correct.
+  const char* name;
+
+  // This specifies the version of the API that is implemented.
+  uint32_t version;
+
+  // The fields below this point were added in version 1 or later.
+
+  // This uses to set max_video_input_size via Starboard Extension.
```

Comment:
We would need more comment on this.  For example (please rephrase):
1. This is used to set the max video input size for any SbPlayer being created later on the calling thread.
2. The max video input size is a hint, the implementation doesn't have to honor it, and there is no feedback.
3. Set it to 0 disable the setting.

Also the comment applies to the Set function, not the Get function.

Reply:
Done.